File: Client/src/main/java/client/dtos/TripDTO.java
package client.dtos;

import model.domain.Trip;

import java.sql.Time;

public class TripDTO {
    private int id;
    private final Trip trip;
    private final String touristAttraction;
    private final String transportCompany;
    private final Time departureTime;
    private final double price;
    private int availableSeats;

    public TripDTO(int id, Trip trip, String touristAttraction, String transportCompany, Time departureTime,
                   double price, int availableSeats) {
        this.id = id;
        this.trip = trip;
        this.touristAttraction = touristAttraction;
        this.transportCompany = transportCompany;
        this.departureTime = departureTime;
        this.price = price;
        this.availableSeats = availableSeats;
    }

    public Trip getTrip() {
        return trip;
    }

    public int getAvailableSeats() {
        return availableSeats;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTouristAttraction() {
        return touristAttraction;
    }

    public String getTransportCompany() {
        return transportCompany;
    }

    public Time getDepartureTime() {
        return departureTime;
    }

    public double getPrice() {
        return price;
    }

    public void setAvailableSeats(int availableSeats) {
        this.availableSeats = availableSeats;
    }
}

File: Client2/utils/TripDTO.cs
using System;

namespace Client2.utils
{
    [Serializable]
    public class TripDTO
    {
        public int Id { get; set; }
        public string TouristAttraction { get; set; }
        public string TransportCompany { get; set; }
        public TimeSpan DepartureTime { get; set; }
        public double Price { get; set; }
        public int AvailableSeats { get; set; }

        public TripDTO(int id, String touristAttraction, String transportCompany, TimeSpan departureTime, double price, int availableSeats)
        {
            Id = id;
            TouristAttraction = touristAttraction;
            TransportCompany = transportCompany;
            DepartureTime = departureTime;
            Price = price;
            AvailableSeats = availableSeats;
        }

        public TripDTO(String touristAttraction, String transportCompany, TimeSpan departureTime, double price, int availableSeats)
        {
            TouristAttraction = touristAttraction;
            TransportCompany = transportCompany;
            DepartureTime = departureTime;
            Price = price;
            AvailableSeats = availableSeats;
        }
    }
}
